docs/jwst/master_background/arguments.rst

```diff
@@ -0,0 +1,18 @@
+.. _msb_step_args:
+
+Step Arguments
+==============
+The master spectroscopic background subtraction step has two optional arguments.
+
+``--user_background``
+  The file name of a user-supplied 1-D master background spectrum. Must be in the form
+  of a standard :ref:`x1d <x1d>` product containing a single 'EXTRACT1D' extension. When set,
+  the creation of a master background spectrum by the step is skipped. The name of the
+  supplied file is recorded in the MSTRBKGD keyword in the output product(s).
+  Defaults to ``None``.
+
+``--save_background``
+  A boolean indicating whether the computed master background spectrum should be saved
+  to a file. Cannot be used when ``user_background`` is specified.
```

Comment:
This seems a little too strong.  I don't think the step should raise an exception if --save_background=True and a user-supplied background was also specified; the step could just print a warning and not save the background.  I think the step currently just ignores save_background if the user supplied a background.

Reply:
fixed in next commit